bazel/examples/debian.BUILD

```diff
@@ -0,0 +1,15 @@
+load("@bazel_tools//tools/build_defs/docker:docker.bzl", "docker_build")
+
+# Extract .xz files
+genrule(
```

Comment:
could we add a custom rule, included with our bazel builder, that let you specify, eg
docker_tar(
  name = "wheezy_tar",
  src = "https://github.com/tianon/docker-brew-debian/tarball/e9bafb113f432c48c7e86c616424cb4b2f2c7a51",
  sha256 = "d31d8215e336a011d9a5594634595c6cb390ef90fa72a51c0043c5e7c3fd5ba1",
)

or something?